Makefile

```diff
@@ -91,32 +91,51 @@ sdk-gitlab:
 
 # simplified runtime for PULP that doesn't need the sdk
 pulp-runtime:
-	git clone https://github.com/pulp-platform/pulp-runtime.git -b v0.0.3
+	git clone https://github.com/pulp-platform/pulp-runtime.git
```

Comment:
You might want to clone a specific version (e.g., a tag or commit hash).  Unless you check out a specific version later in your CI flow, your CI results now depend on the current head of the `master` branch of the `pulp-runtime` repository. When someone pushes a commit to `pulp-runtime`, your CI job could fail even though nothing in your repository has changed since the last successful CI job.

Reply:
Thats a good point (also there reason I put the `-b your-release-tag`). We had this issue back when regression were running on jenkins. You really want your runs to be reproducible.

Reply:
Got it! Thanks!!